jina/peapods/peas/__init__.py

```diff
@@ -159,12 +159,15 @@ def __init__(self, args: 'argparse.Namespace'):
 
     def _set_ctrl_adrr(self):
         """Sets control address for different runtimes"""
-        self._zed_runtime_ctrl_address = self.runtime_cls.get_control_address(
+        self.runtime_ctrl_address = self.runtime_cls.get_control_address(
             host=self.args.host,
             port=self.args.port_ctrl,
             docker_kwargs=getattr(self.args, 'docker_kwargs', None),
         )
 
+        if not self.runtime_ctrl_address:
```

Comment:
Does it ever happen?

Reply:
yes. the new runtime has no ctrl_address and we should use `port_in` instead.

Reply:
ah ok, I did not see is port_in